downstream/modules/platform/ref-operator-ocp-version.adoc

```diff
@@ -4,8 +4,6 @@
 
 [role="_abstract"]
 
-The {OperatorPlatform} to install {PlatformNameShort} {PlatformVers} is available on {OCPShort} 4.9 and later versions.
+The {OperatorPlatform} to install {PlatformNameShort} {PlatformVers} is available on {OCPShort} 4.10 to 4.13.
 
 [role="_additional-resources"]
-.Additional resources
```

Comment:
why are we removing the additional resources section--will the link be updated or replaced by something else?